drivers/flash/spi_nor.c

```diff
@@ -702,6 +702,33 @@ static int spi_nor_read(const struct device *dev, off_t addr, void *dest,
 	return ret;
 }
 
+#if defined(CONFIG_FLASH_EX_OP_ENABLED)
+static int flash_spi_nor_ex_op(const struct device *dev, uint16_t code,
+				const uintptr_t in, void *out)
+{
+	int ret = 0;
+
+	ARG_UNUSED(in);
+	ARG_UNUSED(out);
+
+	acquire_device(dev);
+
+	switch (code) {
+	case FLASH_SPI_NOR_RESET:
+		ret = spi_nor_cmd_write(dev, SPI_NOR_CMD_RESET_EN);
+		if (ret == 0)
+			ret = spi_nor_cmd_write(dev,
+				SPI_NOR_CMD_RESET_MEM);
+		break;
+	default:
+		ret = -ENOTSUP;
+		break;
+	}
+	release_device(dev);
```

Comment:
nit: you added blank lines around `acquire_device`, add some around this as well for coherency